packages/core/src/editor/BlockNoteEditor.ts

```diff
@@ -156,6 +157,11 @@ export type BlockNoteEditorOptions<
     showCursorLabels?: "always" | "activity";
   };
 
+  /**
+   * Options for code blocks.
+   */
+  codeBlock?: CodeBlockOptions;
```

Comment:
minor thought; is it confusing that the term `codeBlock` is used for all of:
- the blocknote block (defined in core)
- the option for the theme / highlighting (defined here)
- the package for advanced theme / highlighting

Reply:
`codeBlock` (this option) makes sense as it is the bundle of options for configuring the code-block block type.
`@blocknote/code-block`(the package) I think could maybe be renamed to `@blocknote/defaults` with an export of `codeBlock` which could make sense if we decide to add more things like this. Or, similar.